docs/apis/options-gateway/introduction-and-setup.md

```diff
@@ -3,99 +3,190 @@ sidebar_position: 0
 ---
 
 # Introduction and Setup
-Panoptic is a protocol that facilitates the trading of [Panoptions](/docs/terms/panoption), which are perpetual options instruments with fixed gamma between two prices that operate on a [streaming premium model](/docs/product/streamia). We have created an [Options Gateway API](https://github.com/panoptic-labs/options_gateway) that facilitates reading information about these markets, as well as managing positions in them. You can use this API to build market making bots, trading bots, liquidation bots, and more.
+Panoptic is a protocol that facilitates the trading of [Panoptions](/docs/terms/panoption), which are perpetual options instruments with fixed gamma between two prices that operate on a [streaming premium model](/docs/product/streamia). Panoptic uses Uniswap as an options clearinghouse and operates with the principles that [LP = Options](/blog/uniswap-lp-equals-options). We have created an [Options Gateway API](https://github.com/panoptic-labs/options_gateway) which provides market data, place orders, and access account information. You can use this API to build market making bots, trading bots, liquidation bots, and more.
```

Comment:
"which provides market data, place orders, and access account information." should be the same tense for each action. Perhaps: "which can provide market data, place orders, and access account information."